app/preference.go

```diff
@@ -9,42 +9,43 @@ import (
 	"github.com/mattermost/mattermost-server/model"
 )
 
-func (a *App) GetPreferencesForUser(userId string) (model.Preferences, *model.AppError) {
-	if result := <-a.Srv.Store.Preference().GetAll(userId); result.Err != nil {
+func (a *App) GetPreferencesForUser(userID string) (model.Preferences, *model.AppError) {
+	result := <-a.Srv.Store.Preference().GetAll(userID)
+	if result.Err != nil {
 		result.Err.StatusCode = http.StatusBadRequest
 		return nil, result.Err
-	} else {
-		return result.Data.(model.Preferences), nil
 	}
+	return result.Data.(model.Preferences), nil
 }
 
-func (a *App) GetPreferenceByCategoryForUser(userId string, category string) (model.Preferences, *model.AppError) {
-	if result := <-a.Srv.Store.Preference().GetCategory(userId, category); result.Err != nil {
+func (a *App) GetPreferenceByCategoryForUser(userID string, category string) (model.Preferences, *model.AppError) {
+	result := <-a.Srv.Store.Preference().GetCategory(userID, category)
+	if result.Err != nil {
 		result.Err.StatusCode = http.StatusBadRequest
 		return nil, result.Err
-	} else if len(result.Data.(model.Preferences)) == 0 {
+	}
+	if len(result.Data.(model.Preferences)) == 0 {
 		err := model.NewAppError("getPreferenceCategory", "api.preference.preferences_category.get.app_error", nil, "", http.StatusNotFound)
 		return nil, err
-	} else {
-		return result.Data.(model.Preferences), nil
 	}
+	return result.Data.(model.Preferences), nil
 }
 
-func (a *App) GetPreferenceByCategoryAndNameForUser(userId string, category string, preferenceName string) (*model.Preference, *model.AppError) {
-	if result := <-a.Srv.Store.Preference().Get(userId, category, preferenceName); result.Err != nil {
+func (a *App) GetPreferenceByCategoryAndNameForUser(userID string, category string, preferenceName string) (*model.Preference, *model.AppError) {
+	result := <-a.Srv.Store.Preference().Get(userID, category, preferenceName)
+	if result.Err != nil {
 		result.Err.StatusCode = http.StatusBadRequest
 		return nil, result.Err
-	} else {
-		data := result.Data.(model.Preference)
-		return &data, nil
 	}
+	data := result.Data.(model.Preference)
+	return &data, nil
 }
 
-func (a *App) UpdatePreferences(userId string, preferences model.Preferences) *model.AppError {
+func (a *App) UpdatePreferences(userID string, preferences model.Preferences) *model.AppError {
```

Comment:
I know golint suggest to replace this, and probably would be something interesting to do in the future, but introducing this change here and now, it will be inconsistent with the rest of the code. We use `Id` instead of the suggested by golint `ID`.